builtin/credential/github/path_config.go

```diff
@@ -109,23 +136,26 @@ func (b *backend) pathConfigRead(ctx context.Context, req *logical.Request, data
 	if err != nil {
 		return nil, err
 	}
-
 	if config == nil {
 		return nil, fmt.Errorf("configuration object not found")
 	}
 
-	config.TTL /= time.Second
-	config.MaxTTL /= time.Second
+	d := map[string]interface{}{
+		"organization": config.Organization,
+		"base_url":     config.BaseURL,
+	}
+	config.PopulateTokenData(d)
 
-	resp := &logical.Response{
-		Data: map[string]interface{}{
-			"organization": config.Organization,
-			"base_url":     config.BaseURL,
-			"ttl":          config.TTL,
-			"max_ttl":      config.MaxTTL,
-		},
+	if config.TTL > 0 {
```

Comment:
Can we add a comment here (like the other PR that this is for BC) and a warning in the response?

Reply:
I'll add the comment. Chris didn't want to change from 204 to 200 so path-help and description in the UI will have to suffice.